src/DynamicForm.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+    <b-form :ref="id" :id="id" class="dynamic-form" novalidate>
+        <dynamic-form-control-section v-for="(section, index) in formMeta.controlSections"
+                                      :key="index"
+                                      :control-section="section"
+                                      @change="change($event, index)">
+        </dynamic-form-control-section>
+        <button v-if="includeSubmitButton"
+                class="btn"
+                :class="disabled? 'btn-secondary' : 'btn-submit'"
```

Comment:
"btn-red" changed to "btn-submit" - this is the only code change to the components.